src/main/java/no/nav/foreldrepenger/mottak/innsending/pdf/FellesSøknadInfoRenderer.java

```diff
@@ -41,41 +42,43 @@ public class FellesSøknadInfoRenderer {
         this.textFormatter = textFormatter;
     }
 
-    public float frilansOpptjening(Frilans frilans, FontAwareCos cos, float y) throws IOException {
+    public float frilansOpptjeningForeldrepenger(Frilans frilans, FontAwareCos cos, float y) throws IOException {
+        // bruker stilles spm om harInntektFraFosterhjem og nyOppstartet kun i fp
+        return frilansOpptjening(frilans, cos, y, true);
+    }
+
+    public float frilansOpptjeningSvangerskapspenger(Frilans frilans, FontAwareCos cos, float y) throws IOException {
+        return frilansOpptjening(frilans, cos, y, false);
+    }
+
+    private float frilansOpptjening(Frilans frilans, FontAwareCos cos, float y, boolean gjelderForeldrepenger) throws IOException {
         if (frilans == null) {
             return y;
         }
-        y = frilans(frilans, cos, y);
-        return y;
-    }
-
-    private float frilans(Frilans frilans, FontAwareCos cos, float y) throws IOException {
         y -= renderer.addLeftHeading(txt("frilans"), cos, y);
         List<String> attributter = new ArrayList<>();
         if (frilans.jobberFremdelesSomFrilans()) {
             addIfSet(attributter, "frilanspågår", textFormatter.dato(frilans.periode().fom()));
         } else {
             attributter.add(txt("frilansavsluttet", textFormatter.dato(frilans.periode().fom())));
         }
-        attributter.add(txt("fosterhjem", jaNei(frilans.harInntektFraFosterhjem())));
-        attributter.add(txt("nyoppstartet", jaNei(frilans.nyOppstartet())));
-        y -= renderer.addLinesOfRegularText(attributter, cos, y);
-        if (!frilans.frilansOppdrag().isEmpty()) {
-            y -= renderer.addLineOfRegularText(txt("oppdrag"), cos, y);
-            var oppdrag = safeStream(frilans.frilansOppdrag())
-                .map(o -> o.oppdragsgiver() + " " + textFormatter.periode(o.periode()))
-                .toList();
-            y -= renderer.addBulletList(INDENT, oppdrag, cos, y);
+        if (gjelderForeldrepenger) {
+            attributter.add(txt("fosterhjem", jaNei(frilans.harInntektFraFosterhjem())));
+            attributter.add(txt("nyoppstartet", jaNei(frilans.nyOppstartet())));
+        }
+        y -= renderer.addLinesOfRegularText(INDENT, attributter, cos, y);
+        if (gjelderForeldrepenger && !frilans.frilansOppdrag().isEmpty()) {
+            y -= renderer.addLineOfRegularText(INDENT, txt("oppdrag"), cos, y);
+            var oppdrag = safeStream(frilans.frilansOppdrag()).map(o -> o.oppdragsgiver() + " " + textFormatter.periode(o.periode())).toList();
+            y -= renderer.addBulletList(INDENT_DOUBLE , oppdrag, cos, y);
             y -= PdfElementRenderer.BLANK_LINE;
-        } else {
-            y -= renderer.addLineOfRegularText(txt("oppdrag") + ": Nei", cos, y);
+        } else if (gjelderForeldrepenger) {
+            y -= renderer.addLineOfRegularText(INDENT, txt("oppdrag") + ": Nei", cos, y);
```

Comment:
Skal ikke dette også være med i SVP?

Reply:
Noen av disse feltene skal ut av fp også, ref Trello-saken. Uklart for meg akkurat hvilke eller om det gjelder alle. Det skjer ikke enda.

Reply:
Ahh, da såg jeg bare feil. Den er god 👍